pants_plugins/sendwave/pants_node/package.py

```diff
@@ -252,21 +250,9 @@ async def node_project_package(field_set: NodeProjectFieldSet) -> BuiltPackage:
     )
 
 
-@rule
-async def node_project_docker(field_set: NodeProjectFieldSet) -> DockerComponent:
-    """Build a node_package target into a DockerComponent.
-
-    This allows files generated by the node process to be included in
-    the docker image.
-    """
-    package = await Get(Digest, NodeProjectFieldSet, field_set)
-    return DockerComponent(sources=package, commands=[])
```

Comment:
I spent some time trying to replace this with built-in Docker components, but after deciding to try leaving it out it seems like it works. I think this is probably because the Docker backend [builds the context automatically based on the target dependencies](https://www.pantsbuild.org/docs/docker#adding-dependencies-to-your-docker_image-targets) but I am not totally sure!